types/jest-image-snapshot/tsconfig.json

```diff
@@ -5,10 +5,7 @@
             "es6",
             "dom"
         ],
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictNullChecks": true,
-        "strictFunctionTypes": true,
+        "strict": true,
```

Comment:
just to make typings for `call` work

Reply:
What in particular doesn't work if don't set these to the expected value?

Reply:
It sounds like you should just set `strictBindCallApply` instead, or not test this case (why would someone use call?)

Reply:
The original library uses `.bind` (see https://github.com/americanexpress/jest-image-snapshot/blob/c3cfd12be15eb92f0213ab91cf2cc071c909bacc/__tests__/index.spec.js#L214..L217)

I think it's fine to have in a test either `.call` or `.bind` to make sure that we can indeed bind those functions.

Reply:
I guess, but, use `strictBindCallApply` instead.

Reply:
Looks like [dtslint currently does not support](https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/dtslint/src/checks.ts#L89-L110) this option
So I can also make PR there, but what the strong reason to avoid `"strict" true` ?

Reply:
strict is a moving target and can change depending on the TS version; the other option isn't supported either but better to have one config that doesn't match than to have more than one.

Reply:
I'd also prefer to avoid using it, as it doesn't seem to be used for any package in this monorepo

Reply:
Strict is used in a few other packages, like here `ramda` so it should be okay to be used:
<img width="321" alt="image" src="https://user-images.githubusercontent.com/22725671/229280583-434e826d-6fcb-4464-8cc2-bee4eeae911f.png">

Reply:
I mean, I guess, but other packages doing it is not always permission to do it again. There's loads of configs in DT that shouldn't be there but are and just haven't been fixed.